.circleci/config.yml

```diff
@@ -3,8 +3,6 @@ jobs:
   build:
     parallelism: 1
     working_directory: ~/daffy_lib
-    environment:
-      - CODECOV_TOKEN: 8e290fba-ea5d-444f-b922-72e013c22f96
```

Comment:
I'll approve but just confused/curious; I copy/pasted this structure from porky and it seems to be in all the other repos, eg https://github.com/Zetatango/porky_lib/blob/master/.circleci/config.yml ?  What am I not getting here?

Reply:
It shouldn't be. I'm going to have to go through the other repos and remove/regenerate the codecov API tokens. We should not have secrets checked into code in source control.